### PR TITLE
feat(watchers): connect with database

### LINKED ISSUE #22: Connect watchers with database
Now they’re using a js object.
Connect them to use a mongo db connection and also a connection to redis

Depends on #8 #7 #33 (for schema definition)

### LINKED ISSUE #89: Also send watcher name to client when sending notifications
Use watchers displayName

### PR DESCRIPTION
**Pull request by negebauer**
Closes #22 
Closes #89

### CODE REVIEW

File: src/models/user/index.js [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
 
 const TOKEN_TTL = 15 * 60; // 15 minutes (specified in seconds)
 
+const schema = new Schema({
+  email: {
+    type: String,
+    required: true,
+    index: true,
+  },
+  secret: Schema.Types.String,
```
Comment (sasalatart) [inline src/models/user/index.js]: Should this be required?

File: src/watchers/run-watchers.js [hunk 2]
```diff
@@ -18,121 +20,20 @@ const LOCAL_ENV = {
...
-  {
-    watcher: 'vtr',
-    snapshot: {},
-  },
-];
-
-async function isRunning(id) {
-  const watcherIsRunning = MOCK_REDIS[id];
+function isRunning(id) {
+  const watcherIsRunning = RUNNING_WATCHERS[id];
```
Comment (sasalatart) [inline src/watchers/run-watchers.js]: If `RUNNING_WATCHERS`'s contents are variable, I wouldn't write it with all caps

File: src/models/user/index.js (hunk 1 above)
Comment (negebauer) [reply src/models/user/index.js]: it is required, but the current code creates it after creating the user. That should be changed

Comment (negebauer) [reply src/models/user/index.js]: changed it to be required and have a default function value

**Pull request merged**